anchore_engine/clients/skopeo_wrapper.py

```diff
@@ -143,16 +143,29 @@ def download_image(fulltag, copydir, user=None, pw=None, verify=True, manifest=N
                 if rc != 0:
                     skopeo_error = SkopeoError(cmd=cmd, rc=rc, out=sout, err=serr)
                     if skopeo_error.error_code != AnchoreError.OSARCH_MISMATCH.name:
-                        raise SkopeoError(cmd=cmd, rc=rc, out=sout, err=serr)                    
+                        raise SkopeoError(cmd=cmd, rc=rc, out=sout, err=serr)
                 else:
                     logger.debug("command succeeded: cmd="+str(cmdstr)+" stdout="+str(sout).strip()+" stderr="+str(serr).strip())
-                    success = True                    
+                    success = True
 
             except Exception as err:
                 logger.error("command failed with exception - " + str(err))
                 raise err
 
             if success:
+                if use_cache_dir and dest_type == 'oci':
+                    # syft expects blobs to be nested inside of the oci image directory. If the --dest-shared-blob-dir skopeo option is used we need to
+                    # provide access to the blobs via a symlink, as if the blobs were stored within the oci image directory
+                    blobs_dir = os.path.join(copydir, "blobs")
+                    if os.path.exists(blobs_dir) and os.path.isdir(blobs_dir):
+                        # if this directory is not empty, there is an issue and we should expect an exception
+                        os.rmdir(blobs_dir)
+
+                    cmd = "ln -s {} ./blobs".format(use_cache_dir)
```

Comment:
why is this using a system command instead of linking with Python?

Reply:
I'll switch over to `os.symlink` 👍